### PR TITLE
GA: set identityId, isLoggedIn custom dimensions

### PR DESCRIPTION
**Pull request by cb372**
## What does this change?

Send the user's ID (extracted from the `GU_U` cookie) and their logged-in-ness to GA as custom dimensions.

Tested locally by copying my `GU_U` cookie from PROD and checking the GA pageview event's custom dimensions.
## What is the value of this and can you measure success?

Moar tracking
## Does this affect other platforms - Amp, Apps, etc?

No
## Screenshots

n/a
## Request for comment

@ajosephides @dominickendrick

### CODE REVIEW

File: common/app/views/fragments/analytics/google.scala.html [hunk 1]
```diff
@@ -22,6 +36,55 @@
...
+            userData = cookieData ? JSON.parse(decodeBase64(cookieData.split('.')[0])) : null;
+        if (userData) {
+            return userData[0];
+        } else {
+            return null;
+        }
+    }
+
+    var identityId = getIdentityIdFromCookie();
+    var isLoggedIn = (identityId !== null);
```
Comment (cb372) [inline common/app/views/fragments/analytics/google.scala.html]: @ajosephides Is this what you intended for the `isLoggedIn` dimension? Just a flag to say whether the cookie is set or not? Seems a little redundant.

Comment (ajosephides) [inline common/app/views/fragments/analytics/google.scala.html]: isLoggedIn is a boolean regarding the users current logged in state and set on a hit level scope. The identityId is set on a user level scope. 
When looking at isLoggedIn it should be examined in conjunction with identityId. Just having identityId logged could give the impression that the user is permanently logged in (see user scope in GA) even though they may not be. 
Subscriptions and Membership have had a separate variables in place to isolate someone that has a login but is not currently logged in.
The approach above gives us a consistent way in which to approach this in Google Analytics across our properties.
Does that make sense?

Comment (cb372) [inline common/app/views/fragments/analytics/google.scala.html]: Oh yeah, I forgot they were different scopes. So this code is correct?

Comment (ajosephides) [inline common/app/views/fragments/analytics/google.scala.html]: 👍

File: common/app/views/fragments/analytics/google.scala.html [hunk 2]
```diff
@@ -5,6 +5,20 @@
 
 (function() {
 
+    @*
```
Comment (dominickendrick) [inline common/app/views/fragments/analytics/google.scala.html]: Luckly this code is already excuted earlier in the javascript loading cycle in https://github.com/guardian/frontend/blob/master/common/app/templates/inlineJS/nonBlocking/getUserData.scala.js and should be available in `window.guardian.config.user` if they are logged in!

Comment (cb372) [inline common/app/views/fragments/analytics/google.scala.html]: Lovely! Thank you.

**Comment by dominickendrick** [comment]
👍

**Pull request merged**